generators/app/templates/docs/syncapi.yaml

```diff
@@ -0,0 +1,83 @@
+openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: Swagger <%= name %>
+  description: Documentation for <%= name %> synchronous api
+  termsOfService: http://swagger.io/terms/
+  contact:
+    name: Swagger API Team
+    email: apiteam@swagger.io
+    url: http://swagger.io
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
```

Comment:
As the generator will be extensively used in GuideSmiths projects, we can customize these values to make reference to our organization. But it's not necessary and I don't even know if it's even desired. Just as an idea.

Reply:
Is it true that Guidesmiths is the main user of this generator but image it being used by any other dev / company, they may not want the guidesmiths name to appear there so i believe that we should keep it as general as possible.
Anyway @UlisesGascon may give us another point of view.

Reply:
Ok! I understand your point. That's because I was not even sure whether my suggestion was correct taking into account that it's an open source project. Let's see what Ulises thinks.

Reply:
I agree with @MatteoDiPaolo. let's be more agnostic 👍